result_specification.md

```diff
@@ -2,10 +2,52 @@
 
 ## Generation
 
-- Generated_Item: Generation Result by the model
-- Ground_truth_Item: The anotated result from the paper
+#### Generation Result
+
+- generated_item: Generation Result by the model
+- ground_truth_item: The anotated result from the paper

```

Comment:
but generated_item and ground_truth_item are not in the JSON below. I don't follow.
I would have something like 
```
prediction: {
light_source: "",
},
ground_truth:{
light_source: "",
}
```
predictions and ground truth should also be in different files